rskj-core/src/main/java/co/rsk/peg/whitelist/WhitelistSupportImpl.java

```diff
@@ -166,4 +166,9 @@ private boolean isLockWhitelistChangeAuthorized(Transaction tx) {
         AddressBasedAuthorizer authorizer = whitelistConstants.getLockWhitelistChangeAuthorizer();
         return authorizer.isAuthorized(tx, signatureCache);
     }
+
+    @Override
+    public void save() {
```

Comment:
tests?

Reply:
Let's create a new PR for Tests. In this way, keep on creating small PRs.